package.json

```diff
@@ -94,9 +94,10 @@
     "eslint-plugin-react": "^6.10.3",
     "jest": "^20.0.4",
     "prettier": "^1.11.1",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "react-test-renderer": "^16.0.0",
+    "react": "^16.3.0",
+    "react-dom": "^16.3.0",
+    "react-lifecycles-compat": "^1.1.3",
```

Comment:
you can upgrade to 2.0 version of this library

Reply:
Even to 3.0 now